### PR TITLE
fix(Dockerfile)!: hugo version 0.115.4, apk add npm, update deps

### PR DESCRIPTION
**Pull request by KellyMerrick**
Fixes https://github.com/go-vela/community/issues/817
- apk add npm

Possible breaking change
- uptick hugo version from v0.101.0 to v0.115.4
- workaround, use plugin parameter `hugo_version: <version>` to downtick to an earlier hugo version

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
 # https://github.com/moby/moby/issues/37345
-ARG HUGO_VERSION=0.101.0
+ARG HUGO_VERSION=0.115.4
 
 ##########################################################################
 ##    docker build --no-cache --target binary -t vela-hugo:binary .     ##
 ##########################################################################
 
-FROM alpine as binary
+FROM alpine@sha256:82d1e9d7ed48a7523bdebc18cf6290bdb97b82302a8a9c27d4fe885949ea94d1 as binary
```
Comment (wass3rw3rk) [inline Dockerfile:14]: let's use version + sha like here https://github.com/go-vela/vela-kubernetes/blob/27beb3a21f2fd9553f92c96e885b86eb17453fb3/Dockerfile#L14

File: Dockerfile [hunk 2]
```diff
@@ -25,13 +25,13 @@ RUN chmod 0700 /bin/hugo
 ##    docker build --no-cache -t vela-hugo:local .    ##
 ########################################################
 
-FROM alpine
+FROM alpine@sha256:82d1e9d7ed48a7523bdebc18cf6290bdb97b82302a8a9c27d4fe885949ea94d1
```
Comment (wass3rw3rk) [inline Dockerfile:28]: let's use version + sha like here https://github.com/go-vela/vela-kubernetes/blob/27beb3a21f2fd9553f92c96e885b86eb17453fb3/Dockerfile#L14

**Review by wass3rw3rk — COMMENTED** [review]
minor tweaks, otherwise looks good

File: Dockerfile (hunk 1 above)
Comment (KellyMerrick) [reply Dockerfile]: fixed

File: Dockerfile (hunk 2 above)
Comment (KellyMerrick) [reply Dockerfile]: fixed

**Review by wass3rw3rk — APPROVED** [review]

**Review by ecrupper — APPROVED** [review]

**Pull request merged**